<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QMainWindow" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Janela</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>Estoque:</string>
    </property>
    <widget class="QTableWidget" name="tabela">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>401</width>
       <height>301</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Jogo:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantidade:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Código:</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preço:</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="media_de_preco">
          <property name="text">
           <string>Media de preço: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_show_media">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="menor_preco">
          <property name="text">
           <string>| menor preco: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_show_menor_preco">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="maior_preco">
          <property name="text">
           <string>| Maior preco: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_show_maior_preco">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>428</width>
       <height>37</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="btn_Ordenar_por_disponibilidade">
        <property name="text">
         <string>Ordenar por disponibilidade.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btn_Ordenar_por_nome">
        <property name="text">
         <string>Ordenar por nome</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Cadastro:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>216</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="nome">
              <property name="text">
               <string>Nome do Jogo:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="codigo">
              <property name="text">
               <string>Código:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="quantidade">
              <property name="text">
               <string>Quantidade:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="preco">
              <property name="text">
               <string>Preço:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="entrada_nome"/>
            </item>
            <item>
             <widget class="QLineEdit" name="entrada_codigo"/>
            </item>
            <item>
             <widget class="QLineEdit" name="entrada_quantidade"/>
            </item>
            <item>
             <widget class="QLineEdit" name="entrada_preco"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Botao">
          <property name="text">
           <string>confirmar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_salvar">
            <property name="text">
             <string>Salvar tabela</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_carregar_arquivo">
            <property name="text">
             <string>Carregar um arquivo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="btn_editar">
            <property name="text">
             <string>Editar item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_excluir">
            <property name="text">
             <string>Excluir item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="relogio_digital">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>hora</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>dia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_dia">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>seg Jan 15</string>
    </property>
   </widget>
   <widget class="QLabel" name="hora">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>hora:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
